src/NuGetUtility/LicenseValidator/LicenseValidationResult.cs

```diff
@@ -7,7 +7,7 @@ public record LicenseValidationResult(string PackageId,
         string? PackageProjectUrl,
         string? License,
         string? Copyright,
-        string? Authors,
+        IEnumerable<string>? Authors,
```

Comment:
can you please make the enumerable non nullable?

Reply:
Oh crap, that alters the json-output to always have a field there even when empty (and breaks some tests), maybe revert it ?

Reply:
I'm not too concerned about the tests breaking, as this is expected when adding new fields. We could change the serialization options to not output empty enumerables... However I did not see that the authors field was already comma seperated in the cached data. I think in that context keeping them as comma separated string is a good option.

Reply:
There are 2 places from cached data ;
NuGet.Packaging.ManifestMetadata.Authors (IEnumerable<string>)
NuGet.Protocol.Core.Types.IPackageSearchMetadata.Authors (string)
Either way we need to transform one of them, I initially picked plain string since it is a little easier to handle

Reply:
Let's go with string then....